apps/docs/pages/guides/getting-started/features.mdx

```diff
@@ -88,7 +88,7 @@ RESTful APIs are autogenerated from your database, without a single line of code
 
 ### Auto-generated GraphQL API
 
-Fast GraphQL APIs using our custom Postgres GraphQL extension. [Docs](/docs/guides/api#graphql-api-overview).
+Fast GraphQL APIs using our custom Postgres GraphQL extension. [Docs](/docs/guides/graphql).
```

Comment:
Probably want to link to `/docs/guides/graphql/api` though?

Reply:
Yes, that link is better, updated now. Thanks.